### DISCUSSION TITLE
Is this the right place for students

### DISCUSSION THREAD

**Discussion by Runner3131** [community/community#20694] [GitHub Education] [answered]
Hello,
I was just wondering, because I was told they moved the Github student Education benefits to a different place. This was just for everyone who needed to learn github or something; not the same education for us students.
Was I told wrong,
Or if I'm right, where is the student community located?

Thank you very much.

Tim J
XxFreeLancerxX

**Comment by melvincwng** [comment, marked as answer]
Hey @Runner3131, first of all, let me welcome to GitHub Community 👋.

Just to clarify, yup, you are at the right place! There used to be a separate forum for GitHub Education Community. But as of Apr 2022, it has been moved to here.

Feel free to add ask questions here in this discussion forum, if you face any technical or non technical related issues.

You can refer to this post [here](https://github.community/t/the-github-education-community-category-has-moved/242888) for more info.

Hope this clarifies.

**Reply by melvincwng** [reply]
@Runner3131 , to add on, if you need more information on what benefits GitHub Student Developer Pack provides, please refer to this [link](https://education.github.com/pack/offers)
